backend/src/api/api.ts

```diff
@@ -1,10 +1,11 @@
 import { FastifyPluginAsync } from 'fastify';
 import { ApiPath } from '~/common/enums/enums';
-import { master, token, tenant } from '~/services/services';
+import { master, token, tenant, group } from '~/services/services';
 import { initMastersApi } from './masters/masters.api';
 import { initTenantsApi } from './tenants/tenants.api';
 import { authorization as authorizationPlugin } from '~/plugins/plugins';
 import { WHITE_ROUTES } from '~/common/constants/constants';
+import { initGroupApi } from '~/api/groups/groups.api';
```

Comment:
should be part of eam api
```suggestion
import { initEAMApi } from '~/api/eam/eam.api';
```